draft-ietf-tsvwg-udp-ecn.md

```diff
@@ -104,160 +104,166 @@ are likely to have different syntax and capabilities.
 
 # Conventions and Definitions
 
-{::boilerplate bcp14-tagged}
-
 This document is not a general tutorial on UDP socket programming, and assumes
 familiarity with basic socket concepts like binding, socket options, and
 common system error codes.
 
-Throughout this document, "Apple" refers to both MacOS and iOS.
+Throughout this document, "Apple" refers to both macOS and iOS.
 
-# Receiving ECN codepoints
+# Receiving the ECN field
 
-Network devices can change the ECN codepoint in the IP header. Since this
+Network devices can change the ECN field in the IP header. Since this
```

Comment:
```suggestion
Network devices can change the ECN codepoint in the IP header. Since this
```

Reply:
Thank you. I am fine with a separate PR replacing marks and flags with codepoint.